static/js/libs/mouse.js

```diff
@@ -0,0 +1,18 @@
+class Mouse {
+  constructor() {
+    this.position = {x: 0, y: 0};
+
+    window.addEventListener('mousemove', this.updatePosition.bind(this));
```

Comment:
Not sure if it part of this PR, but do we need a global mouse object that always listens to mouse move on whole window? Instead of some event handling inside particular module that needs this?

Reply:
All the graphs will use the mouse to position the tooltips, one event handler is more efficient than multiple - iirc... I could be wrong?